test/e2e/common.go

```diff
@@ -56,13 +57,12 @@ var (
 		agentFlavorDatadogIOTAgent:  "datadog.yaml",
 	}
 	osConfigByPlatform = map[string]osConfig{
-		"Debian_11":         {osType: ec2os.DebianOS},
-		"Ubuntu_22_04":      {osType: ec2os.UbuntuOS},
-		"RedHat_CentOS_6":   {osType: ec2os.CentOS, ami: "ami-0506f01ccb6dddeda"},
-		"RedHat_CentOS_7":   {osType: ec2os.CentOS},
-		"RedHat_8":          {osType: ec2os.RedHatOS, ami: "ami-06640050dc3f556bb"},
-		"Amazon_Linux_2023": {osType: ec2os.AmazonLinuxOS, ami: "ami-0889a44b331db0194"},
-		"openSUSE_15":       {osType: ec2os.SuseOS},
+		"Debian_12":         {osDescriptor: componentsos.DebianDefault},
+		"Ubuntu_22_04":      {osDescriptor: componentsos.UbuntuDefault},
+		"RedHat_CentOS_9":   {osDescriptor: componentsos.CentOSDefault},
+		"RedHat_8":          {osDescriptor: componentsos.NewDescriptor(componentsos.RedHat, "8"), ami: "ami-06640050dc3f556bb"},
+		"Amazon_Linux_2023": {osDescriptor: componentsos.AmazonLinux2, ami: "ami-0889a44b331db0194"},
+		"openSUSE_15":       {osDescriptor: componentsos.SuseDefault},
```

Comment:
I removed CentOS 6 because we indeed no longer support it. For the other I updated the version to reflect the version that are now provided by the framework out of the box.
If we want to keep testing the same version some extra work would be required, but not sure this is really relevant since, for example debian 11 was not the oldest version we support

Reply:
I just pushed a commit to try to keep the same versions, let's see if it works

Reply:
All good, I was mostly curious - or checking that we were intentionally making the change. Feel free to keep the change in place!